client/src/unifycr-sysio.c

```diff
@@ -358,6 +358,7 @@ int UNIFYCR_WRAP(__xstat)(int vers, const char *path, struct stat *buf)
     }
 }
 
+#if 0
```

Comment:
I suggest deleting these code blocks. Keeping them commented out hurts readability by adding clutter, and it's not necessary since it can always be retrieved from the git history if needed. If we plan to re-enable this later let's open a github issue to track that work.